apps/frontend/src/app/Data/Characters/Somnia/index.tsx

```diff
@@ -0,0 +1,680 @@
+        },
+        naught
+      )
+    )
+  ),
+  { name: ct.ch('c1Key'), unit: '%', asConst: true }
+)
+
+const [condC2PrimePath, condC2Prime] = cond(key, 'c2Prime')
+const c2PrimeDmgInc = greaterEq(
+  input.constellation,
+  2,
+  equal(
+    condSuperposition,
+    'on',
+    equal(condC2Prime, 'on', { ...superposition_normCharged_dmgInc })
+  )
+)
+
+const [condC6StacksPath, condC6Stacks] = cond(key, 'c6Stacks')
+const c6StacksArr = range(1, dm.constellation6.maxStacks)
+const c6_critRate_ = greaterEq(
+  input.constellation,
+  6,
+  lookup(
+    condC6Stacks,
+    objectKeyMap(c6StacksArr, (stack) =>
+      prod(percent(dm.constellation6.critRate_), stack)
+    ),
+    naught
+  )
+)
+const c6_critDMG_ = greaterEq(
+  input.constellation,
+  6,
+  lookup(
+    condC6Stacks,
+    objectKeyMap(c6StacksArr, (stack) =>
+      prod(percent(dm.constellation6.critDMG_), stack)
+    ),
+    naught
+  )
+)
+
+const dmgFormulas = {
+  normal: Object.fromEntries(
+    dm.normal.hitArr.map((arr, i) => [
+      i,
+      dmgNode('atk', arr, 'normal', undefined, c1ModeMult),
+    ])
+  ),
+  charged: {
+    dmg: dmgNode('atk', dm.charged.dmg, 'charged', undefined, c1ModeMult),
+  },
+  plunging: Object.fromEntries(
+    Object.entries(dm.plunging).map(([key, value]) => [
+      key,
+      dmgNode('atk', value, 'plunging'),
+    ])
+  ),
+  skill: {
+    pressDmg: dmgNode('atk', dm.skill.dmg, 'skill'),
+    normal_dmgInc: superposition_normCharged_dmgInc,
+    charged_dmgInc: { ...superposition_normCharged_dmgInc },
+  },
+  burst: {
+    initialDmg: dmgNode('atk', dm.burst.pressDmg, 'burst'),
+    multDotDmg: dmgNode('atk', dm.burst.multDotDmg, 'burst'),
+    addDotDmg: dmgNode('atk', dm.burst.addDotDmg, 'burst'),
+    subDotDmg: dmgNode('atk', dm.burst.subDotDmg, 'burst'),
+    sunderDmg: dmgNode('eleMas', dm.burst.sunderDmg, 'burst'),
+    supernovaDmg: dmgNode('eleMas', dm.burst.supernovaDmg, 'burst'),
+    eleMas: equal(
+      condCycloneActive,
+      'on',
+      prod(
+        subscript(input.total.burstIndex, dm.burst.eleMas_, { unit: '%' }),
+        input.premod.eleMas
+      )
+    ),
+    sub_normal_dmgInc: subtraction_normCharged_dmgInc,
+    sub_charged_dmgInc: { ...subtraction_normCharged_dmgInc },
+    ...objectKeyMap(allElementWithPhyKeys, (_) => ({ ...lessThan3_eleRes_ })),
+  },
+  passive2: {
+    eleMas: greaterEq(
+      input.asc,
+      4,
+      equal(
+        condCycloneActive,
+        'on',
+        lookup(
+          condA4EnemiesHit,
+          objectKeyMap(a4EnemiesArr, (stack) =>
+            prod(percent(dm.passive2.elemas), stack, input.premod.eleMas)
+          ),
+          naught
+        )
+      )
+    ),
+  },
+  constellation2: {
+    normal_dmgInc: c2PrimeDmgInc,
+    charged_dmgInc: { ...c2PrimeDmgInc },
+  },
+}
+
+const burstC3 = greaterEq(input.constellation, 3, 3)
+const skillC5 = greaterEq(input.constellation, 5, 3)
+
+const data = dataObjForCharacterSheet(
+  key,
+  elementKey,
+  'fontaine',
```

Comment:
Just verifying, we will keep her Fontaine, and update her card art whenever a real fontaine char comes?

Reply:
Another option is to keep her as "Monstat"(to match the cover) and then change it when we know more about fontaine